exporter/collectors/cpu.py

```diff
@@ -11,28 +11,34 @@ def __init__(self) -> None:
 
     def get_utilization(self):
         """
-        TODO: @
+        @mccooeyc11
         Returns the CPU utilization as a percentage.
         """
-        pass
+        return psutil.cpu_percent()
 
-    def get_frequency():
+    def get_frequency(self):
         """
-        TODO: @
+        @mccooeyc11
         Returns the current CPU frequency in MHz.
         """
-        return 0    # TODO: here for CI Proof of Concept
+        return psutil.cpu_freq()[0]
 
     def get_temperature(self):
         """
-        TODO: @
+        TODO: @mccooeyc11
         Returns the CPU temperature in degrees Celsius.        
         """
-        pass
+        if not psutil.sensors_temperatures():
+            return 20.0
+        else:
+            return psutil.sensors_temperatures()[0][0][1]
```

Comment:
### Requested changes
* It's good to check if sensors are None but we should not be returning an arbitrary value like 20.0
* What I would suggest instead is to expand the logic with something like this:
```
# pseudo code

if psutil.sensors_temperatures() is None
      return None

get 'coretemp'

if coretemp has > 0 sensors
      return the first sensor
else return None
```
* I would test the structure of the data by printing stuff to the console of the docker container:
![image](https://github.com/max-romagnoli/System-Telemetry-Agent/assets/68606608/9e612e36-fdee-4fc9-8b12-6511caaebbf2)